pkg/utils/reader.go

```diff
@@ -72,6 +89,91 @@ func InitFileReader() *FileReader {
 	return &FileReader{}
 }
 
+type msg struct {
+	Msg        string
+	City       string
+	Country    string
+	Latency    int
+	Batch      string
+	FirstName  string
+	LastName   string
+	Gender     string
+	Hostname   string
+	HTTPStatus int
+	Hobby      string
+}
+
+var logMessages = []string{
+	"%s for DRA plugin %q failed. Plugin returned an empty list for supported versions",
+	"%s for DRA plugin %q failed. None of the versions specified %q are supported. err=%v",
+	"Unable to write event '%v' (retry limit exceeded!)",
+	"Unable to start event watcher: '%v' (will not retry!)",
+	"Could not construct reference to: '%v' due to: '%v'. Will not report event: '%v' '%v' '%v'",
+}
+
+func createK8sBody(logEntry *msg, f *gofakeit.Faker) *msg {
+	logEntry.Batch = fmt.Sprintf("batch-%d", f.Number(1, 1000))
+	logEntry.FirstName = f.FirstName()
+	logEntry.LastName = f.LastName()
+	logEntry.Gender = f.Gender()
+	logEntry.Hostname = f.IPv4Address()
+	logEntry.HTTPStatus = f.Number(200, 500)
+	logEntry.City = f.City()
+	logEntry.Country = f.Country()
+	logEntry.Latency = f.Number(0, 100)
+	logEntry.Hobby = f.Hobby()
```

Comment:
you can just put this line `logEntry.Msg = replacePlaceholders(randomTemplate)` here and then you will not need the `randomizeLogEntry` function.